.tool-versions

```diff
@@ -0,0 +1,2 @@
+erlang 20.2
+elixir v1.5.3-otp-20
```

Comment:
Curious why these specific versions. Are we using the same ones on other projects? (I'm def in favor of needing as few distinct OTP versions as possible, since they take forever to install)

Reply:
Not sure. These are the versions that were in my local `.tool-versions` when I came back to this repo.

Reply:
And given that the previous commit on this repo was in 2019, I'm not super motivated to do any updates here :)